config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -11,6 +11,41 @@ presubmits:
       preset-service-account: "true"
     optional: true
     always_run: true
+    path_alias: k8s.io/kubernetes
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20210224-a27864c-master
+          command:
+            - runner.sh
+            - bash
+            - -c
+          args:
+            # TODO: stop depending on custom timeout
+            - 'make test KUBE_RACE=-race KUBE_TIMEOUT=--timeout=240s'
+          # TODO: direct copy from pull-kubernetes-bazel-test, tune these
+          resources:
+            limits:
+              cpu: 4
+              memory: "36Gi"
+            requests:
+              cpu: 4
+              memory: "36Gi"
+  - name: pull-kubernetes-unit-experimental
+    # try clonerefs with preclone
+    decoration_config:
+      utility_images:
+        clonerefs: "gcr.io/spiffxp-gke-dev/clonerefs-k8s-preclone:v20210225-6252cf69ff"
```

Comment:
seems legit

/lgtm
/approve

Reply:
we need a staging repo, and I'm a bit hung up on whether we should:
- keep using per-subproject (`staging-prow`, `staging-test-infra`, etc.)
- or consolidate as some other sigs have done `staging-sig-testing`

so I opted to unblock us since my repo is world readable for now